code/_onclick/item_attack.dm

```diff
@@ -1,74 +1,65 @@
-
 // Called when the item is in the active hand, and clicked; alternately, there is an 'activate held object' verb or you can hit pagedown.
 /obj/item/proc/attack_self(mob/user)
 	return
-
+	
 // No comment
 /atom/proc/attackby(obj/item/W, mob/living/user)
 	return
+	
 /atom/movable/attackby(obj/item/W, mob/living/user)
 	if(W)
 		if(!(W.flags_item & NOBLUDGEON))
 			visible_message("<span class='danger'>[src] has been hit by [user] with [W].</span>", null, 5)
 			user.animation_attack_on(src)
 			user.flick_attack_overlay(src, "punch")
-
+			
 /mob/living/attackby(obj/item/I, mob/user)
 	if(istype(I) && ismob(user))
 		I.attack(src, user)
-
-
+		
 // Proximity_flag is 1 if this afterattack was called on something adjacent, in your square, or on your person.
 // Click parameters is the params string from byond Click() code, see that documentation.
 /obj/item/proc/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
 	return
-
-
+	
 /obj/item/proc/attack(mob/living/M, mob/living/user, def_zone)
 	if(flags_item & NOBLUDGEON)
 		return
-
 	if (!istype(M)) // not sure if this is the right thing...
 		return 0
-
 	if (M.can_be_operated_on())        //Checks if mob is lying down on table for surgery
 		if (do_surgery(M,user,src))
 			return 0
-
 	/////////////////////////
 	user.lastattacked = M
 	M.lastattacker = user
-
 	log_combat(user, M, "attacked", src, "(INTENT: [uppertext(user.a_intent)]) (DAMTYE: [uppertext(damtype)])")
 	msg_admin_attack("[key_name(user)] attacked [key_name(M)] with [name] (INTENT: [uppertext(user.a_intent)]) (DAMTYE: [uppertext(damtype)])" )
-
 	/////////////////////////
-
 	add_fingerprint(user)
-
 	var/power = force
+	var/bonus = 0
 	if(HULK in user.mutations)
 		power *= 2
-
 	if(user.mind && user.mind.cm_skills)
-		power = round(power * (1 + 0.3*user.mind.cm_skills.melee_weapons)) //30% bonus per melee level
-
+		bonus += round(power * 0.3 * max(0,user.mind.cm_skills.melee_weapons)) //30% bonus per melee level
+	if(ishuman(user))
+		var/mob/living/carbon/human/A = user
+		if(A.command_aura == "move") //Check for move aura; improves melee damage significantly.
+			bonus += 5 + round(power * max(0, A.command_aura_strength) * 0.20) // No cumulative multiplication with melee bonus. Move aura adds damage = 5 + 20% of base * Command level - 1, or +5 and +60% for a CO/XO.
```

Comment:
Telebaton has `force = 50`.
Add 60% from old synth melee bonus and that gets +30
Add 5 + 60% and that gets +35.
Total damage: 125 brute per hit. Which bypasses armor.

Seems a bit excessive.

Reply:
Seems like that's more an issue with synths.

Also it's pretty much nothing compared to again, PB weapons, but I'll filter synths out though because they shouldn't be benefiting from orders as they're machines incapable of real emotion/inspiration.